common/startup/application_startup.c

```diff
@@ -163,10 +163,10 @@ static void repeated_timer_handler(void)
         inactivity_counter = 0;
         if (counter.level > LEVEL_ONE) {
             mark_error_screen(ui_text_process_reset_due_to_inactivity);
-            reset_flow_level();
-            lv_obj_clean(lv_scr_act());
             if (CY_External_Triggered())
                 comm_reject_request(STATUS_PACKET, STATUS_CMD_ABORT);
+            reset_flow_level();
```

Comment:
This fix might be needed in other areas where reset flow level is called before external triggered check.
Should identify cases and add those too.

Reply:
verified. no other such instance.